### ISSUE TITLE
Contrast ratio is low for Tag component.

### ISSUE THREAD

**Issue by Shike-m** [SAP/ui5-webcomponents#11531]
### Describe the bug

When the "design" property is set to "Negative" in the "Tag" component, the contrast ratio is "3.98:1", which fails to pass the WCAG test for "normal text" contrast.

### Isolated Example

Check the storybook: https://sap.github.io/ui5-webcomponents-react/v2/?path=/docs/data-display-tag--docs

### Reproduction steps

1. In the storyBook: https://sap.github.io/ui5-webcomponents-react/v2/?path=/docs/data-display-tag--docs.
2. Select "Negative" for the "design" property.
3. Check the Tag example's background color is rgb(245,50,50), and text color is white, rgb(255,255,255);
4. Transfer the rgb to hex and check the contrast ratio: https://webaim.org/resources/contrastchecker/

### Expected Behaviour

Pass the checker test, there is a customer case which mentions the contrast radio is too low. 

### UI5 Web Components for React Version

2.9.0

### UI5 Web Components Version

2.9.0

### Browser

Chrome

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Valkanister** [comment]
Hello @Shike-m, 

The color contrast is not correctly measured. The text in the "Negative" tag has a text shadow around it. You should measure against this darker color. 

Kind regards, 

Georgi

SAPUI5 | SAP Development Support
